evadb/third_party/databases/interface.py

```diff
@@ -26,11 +26,16 @@ def get_database_handler(engine: str, **kwargs):
     their new integrated handlers.
     """
 
-    # Dynamically install dependencies.
-    dynamic_install(engine)
-
     # Dynamically import the top module.
-    mod = dynamic_import(engine)
+    try:
+        mod = dynamic_import(engine)
+    except ImportError:
+        req_file = os.path.join(
+            "evadb", "third_party", "databases", engine, "requirements.txt"
```

Comment:
Is this path accessible if user directly installs from pip instead of source?

Reply:
Yeah, I realize this path will only work for dev testing. We should switch using absolute path. In addition, we also need to package `requirements.txt`.

Reference to my closed PR: https://github.com/georgia-tech-db/evadb/pull/1001/files

Reply:
Maybe just add a new property function to the handler that returns the list of package. No need to worry about path .

Reply:
We won't be able to import the class to access the property function.